app/assets/javascripts/models/beta/projectBoard.js

```diff
@@ -3,13 +3,21 @@ import changeCase from 'change-object-case';
 import * as Story from './story';
 import * as Project from './project';
 
-export function get(projectId) {
-  return httpService
-    .get(`/beta/project_boards/${projectId}`)
-    .then(({ data }) => changeCase.camelKeys(data, { recursive: true, arrayRecursive: true }))
-    .then((projectBoard) => ({
-      ...projectBoard,
-      project: Project.deserialize(projectBoard),
-      stories: projectBoard.stories.map(Story.deserialize)
-    }));
+export const get = async (projectId) => {
+  const { data } = await httpService
+    .get(`/beta/project_boards/${projectId}`);
+
+  const projectBoard = deserialize(data);
+
+  return {
+    ...projectBoard,
+    project: Project.deserialize(projectBoard),
```

Comment:
Should this be `Project.deserialize(projectBoard.project)`?

Reply:
No, it uses the `board` and picks the labels that are inside it, and return it with the `board.project`.